_generated/CondaAuthenticateV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,8 +3,6 @@
   "loc.helpMarkDown": "",
   "loc.description": "Authentication task for the conda client",
   "loc.instanceNameFormat": "Conda Authenticate",
-  "loc.input.label.workloadIdentityServiceConnection": "'Azure DevOps' Service Connection",
-  "loc.input.help.workloadIdentityServiceConnection": "If this is set, it will generate credentials using service connection instead of the build pipeline identity.",
   "loc.messages.FailedToAddAuthentication": "Failed to add authentication.",
```

Comment:
This change seems incorrect. I see these strings in [Tasks/CondaAuthenticateV0/Strings/resources.resjson/en-US/resources.resjson](https://github.com/microsoft/azure-pipelines-tasks/blob/9b57540c52ecc3a8c80257abcb06ec5b56f9dbb2/Tasks/CondaAuthenticateV0/Strings/resources.resjson/en-US/resources.resjson#L6-L7). Please revert this file.